lib/Slider/Slider.js

```diff
@@ -579,6 +586,10 @@ module.exports = kind(
 			this.updatePopup(this.elasticFrom);
 			this.set('value',this.elasticFrom);
 
+			if (this.enableJumpIncrement) {
+				this.updateButtonStatus(v);
```

Comment:
I was curious about whether we should only update the button status when dragging has finished, as opposed to checking/updating while dragging. I spoke with @Djspaceg and Robert about the UX here and they are indifferent to the behavior (so as long as the buttons are updated after dragging has finished). As a result, I think we should go with the more performant path and only update the button status when dragging has finished.